src/main/scala/org/zalando/spearheads/innkeeper/api/JsonProtocols.scala

```diff
@@ -80,10 +96,30 @@ object JsonProtocols extends DefaultJsonProtocol {
     }
   }
 
-  implicit val pathMatcherFormat = jsonFormat(PathMatcher.apply, "matcher", "matcher_type")
+  implicit val pathMatcherFormat = jsonFormat(PathMatcher.apply, "match", "type")
+
+  val newRouteFormat = jsonFormat(NewRoute.apply, "description", "match_path", "endpoint",
+    "match_headers", "match_methods", "request_headers", "response_headers", "path_rewrite")
 
-  implicit val newRouteFormat = jsonFormat(NewRoute.apply, "description", "path_matcher", "endpoint",
-    "header_matchers", "method_matchers", "request_headers", "response_headers", "path_rewrite")
+  implicit object NewRouteFormat extends RootJsonFormat[NewRoute] {
+
+    override def write(obj: NewRoute): JsValue = newRouteFormat.write(obj)
+
+    override def read(json: JsValue): NewRoute = {
+      val newRoute = newRouteFormat.read(json)
+
+      NewRoute(
+        description = newRoute.description,
+        pathMatcher = newRoute.pathMatcher,
+        endpoint = newRoute.endpoint,
+        headerMatchers = newRoute.headerMatchers.orElse(Some(Seq.empty)),
+        methodMatchers = newRoute.methodMatchers.orElse(Some(Seq("GET"))),
+        requestHeaders = newRoute.requestHeaders.orElse(Some(Seq.empty)),
+        responseHeaders = newRoute.responseHeaders.orElse(Some(Seq.empty)),
+        pathRewrite = newRoute.pathRewrite
```

Comment:
same as previous comment. I don't like the duplication of the defaults. It might lead to subtle bugs when the defaults change in only one place